spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    email { "user@example.com" }
```

Comment:
unique制約のあるカラムは`sequence`を設定しましょう！
```suggestion
    sequence(:email) {|n| "user_#{n}@example.com"}
```

Reply:
2c57945